src/redux/sagas/manualUploadFormsSaga.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  all, call, put, takeLatest,
+} from 'redux-saga/effects';
+import apiClient from '../../utils/apiClient';
+import { sessionToken } from '../reducers/tokenReducer';
+import isAuthenticated from '../../utils/authHelper';
+import { IToken } from '../../models/token/IToken';
+import { refreshToken } from './tokenSaga';
+import { IManualUploadFormsType } from '../../models/manualUploadForms/IManualUploadForms';
+import { getManualUploadFormsFailure, getManualUploadFormsSuccess } from '../actions/menuUpload/menuUploadActions';
+import ManualUploadActionTypes from '../actions/menuUpload/menuUploadTypes';
+
+const getManualUploadForms = async () => {
+  const accessToken = JSON.parse(localStorage.getItem(sessionToken) as string).access;
+  const { data } = await apiClient.get(
+    'available_forms/',
+    {
+      headers: {
+        'Content-type': 'application/json',
+        Authorization: `Bearer ${accessToken}`,
+      },
+    },
+  );
+
+  return data;
+};
+
+function* getManualUploadFormsSaga() {
+  try {
+    if (isAuthenticated() === false) {
```

Comment:
@KarolinaKopacz I will fix this while working on delete because this part of code is reusable in many places. But in that PR scope we can merge. (I will have this fixed in my PR that is going to be created - I moved this refresh part into saga utils and this will be reusable).